integrations/tensorflow/python_projects/iree_tf/iree/tools/tf/.gitignore

```diff
@@ -0,0 +1 @@
+iree-import-tf
```

Comment:
Can we put the binaries somewhere else instead. Having to have them in the source tree strikes me as quite weird

Reply:
Nope, they have to go here (or at least the symlinks do). We could add a build step to the Python packages, but that increases the complexity by a lot and will be quite a bit more fragile.